aikau/src/main/resources/alfresco/services/DocumentService.js

```diff
@@ -208,7 +197,12 @@ define(["dojo/_base/declare",
             }
             else
             {
-               url = AlfConstants.URL_SERVICECONTEXT + "components/documentlibrary/data/node/" + targetNodeUri + params;
+               url = AlfConstants.URL_SERVICECONTEXT + "components/documentlibrary/data/";
+               if (payload.site)
+               {
+                   url += 'site/' + encodeURIComponent(payload.site) + '/';
```

Comment:
Could you change the single quotes to be double quotes please to avoid JSHint errors on out automated builds